.github/workflows/merge_to_main_production.yaml

```diff
@@ -48,6 +52,12 @@ jobs:
         run: |
           kubectl apply -k env/production --kubeconfig=/home/runner/.kube/config
 
+      - name: Deploy lambda
+        run: |
+          aws lambda update-function-code \
+            --function-name api-lambda \
+            --image-uri $PRIVATE_ECR/$API_LAMBDA_IMAGE
```

Comment:
Sorry for late reply... Would that not make more sense to have this with the terraform infrastructure version release? As this will be part of AWS? We normally don't have pieces of AWS in this repository.

Reply:
For the long term, ie when it goes into real production, what I like about having it here (with auto-PRs) is that
- it keeps the k8s api and lambda api in sync (useful during canary release, though not if the k8s api is gone)
- it (the pr-bot PR) gives us insight into what changes are going into it

For a shorter term, we could just make PRs to the terraform repo as you suggest.

thoughts?

Reply:
I think you bring a valid point with having both at the same place to have these in sync. We could push what we have for now and reflect back. It would be nice to have an easier process all at the same place (infra, k8n, app). It's just a bit weird to TF dependent release with k8s exclusive repository. All good for now!